### PR TITLE
Claudia 6268 installation manual

### PR DESCRIPTION
**Pull request by hmunfru**
REVIEWERS

@jesuspg @flopezag 

DESCRIPTION

Installation manual

TESTING

Documentation

### CODE REVIEW

File: doc/admin.md [hunk 1]
```diff
@@ -49,4 +49,280 @@ Finally, we execute the commando package-import for the murano client library (c
 
 It will update the Murano core library with the new FIWARE extensions.
 
+## Sanity check procedures
+
+The Sanity Check Procedures are the steps that a System Administrator
```
Comment (flopezag) [inline doc/admin.md]: procedures

File: doc/admin.md [hunk 2]
```diff
@@ -49,4 +49,280 @@ Finally, we execute the commando package-import for the murano client library (c
...
+therefore a preliminary set of tests to ensure that obvious or basic
+malfunctioning is fixed before proceeding to unit tests, integration
+tests and user validation.
+
+### End to End testing
+
+Although one End to End testing must be associated to the Integration
+Test, we can show here a quick testing to check that everything is up
+and running. For this purpose we send a request to our API in order to
+test the credentials that we have from then and obtain a valid token to
```
Comment (flopezag) [inline doc/admin.md]: from what?

This sentece seems that you test the Keyrock not Murano.

File: doc/admin.md [hunk 3]
```diff
@@ -49,4 +49,280 @@ Finally, we execute the commando package-import for the murano client library (c
...
+work with.
+
+In order to make a probe of the different functionalities related to the
+Murano, we start with the obtention of a valid token for a
+registered user. Due to all operations of the Murano are using
+the security mechanism which is used in the rest of the cloud component,
```
Comment (flopezag) [inline doc/admin.md]: mechanism, which

File: doc/admin.md [hunk 4]
```diff
@@ -49,4 +49,280 @@ Finally, we execute the commando package-import for the murano client library (c
...
+
+Although one End to End testing must be associated to the Integration
+Test, we can show here a quick testing to check that everything is up
+and running. For this purpose we send a request to our API in order to
+test the credentials that we have from then and obtain a valid token to
+work with.
+
+In order to make a probe of the different functionalities related to the
+Murano, we start with the obtention of a valid token for a
+registered user. Due to all operations of the Murano are using
```
Comment (flopezag) [inline doc/admin.md]: Due to all Murano operations

File: doc/admin.md [hunk 5]
```diff
@@ -49,4 +49,280 @@ Finally, we execute the commando package-import for the murano client library (c
...
+the security mechanism which is used in the rest of the cloud component,
+it is needed to provide a security token in order to continue with the
```
Comment (flopezag) [inline doc/admin.md]: the rest of operations -> them

File: doc/admin.md [hunk 6]
```diff
@@ -49,4 +49,280 @@ Finally, we execute the commando package-import for the murano client library (c
...
+rest of operations. For this operation we need to execute the following
```
Comment (flopezag) [inline doc/admin.md]: operation -> purpose

File: doc/admin.md [hunk 7]
```diff
@@ -49,4 +49,280 @@ Finally, we execute the commando package-import for the murano client library (c
...
+curl sentence.
+
+    curl -d '{"auth": {"tenantName": $TENANT,
+    "passwordCredentials":{"username": $USERNAME, "password": $PASSWORD}}}'
+    -H "Content-type: application/json" -H "Accept: application/xml"
+    http://$KEYSTONE_IP:$KEYSTONE_PORT/v2.0/tokens
+
+Both $TENANT (Project), $USERNAME and $PASSWORD must be values
+previously created in the OpenStack Keystone. The $KEYSTONE_IP and $KEYSTONE_PORT are the data of our internal
+installation of IdM (Keystone).
```
Comment (flopezag) [inline doc/admin.md]: not write IdM just Keystone or Keyrock

File: doc/admin.md [hunk 8]
```diff
@@ -49,4 +49,280 @@ Finally, we execute the commando package-import for the murano client library (c
...
+
+After it, we can check if Murano is up and running with a
+single instruction which is used to return the information of the status
+of the processes together with the queue size.
+
+    curl -v -H 'X-Auth-Token: a9a861db6276414094bc1567f664084d'
+    -X GET http://murano.lab.fiware.org:8082
+
+This operation will return the information regarding the tenant details
+of the execution of the Policy Manager
```
Comment (flopezag) [inline doc/admin.md]: Well too much copy/paste... rewrite it...

File: doc/admin.md [hunk 9]
```diff
@@ -49,4 +49,280 @@ Finally, we execute the commando package-import for the murano client library (c
...
+                       "href": "http://murano.lab.fiware.org:8082/v1/",
+                       "rel": "self"
+                   }
+               ]
+           }
+       ]
+    }
+
+
+For more details to use this GE, please refer to the User & Programmers Guide.
```
Comment (flopezag) [inline doc/admin.md]: Link to the User & Programmers Guide... asume the oficial OpenStack Murano web site.

File: doc/admin.md [hunk 10]
```diff
@@ -49,4 +49,280 @@ Finally, we execute the commando package-import for the murano client library (c
...
+    root       762   739  0 10:50 ?        00:00:02 /opt/murano/.tox/venv/bin/python .tox/venv/bin/murano-api --config-file etc/murano/murano.conf
+    root       800     5  0 10:50 ?        00:00:00 /usr/bin/python /usr/local/bin/tox -e venv -- murano-engine --config-file ./etc/murano/murano.conf
+    root       821   800  0 10:50 ?        00:00:02 /opt/murano/.tox/venv/bin/python .tox/venv/bin/murano-engine --config-file etc/murano/murano.conf
+
+In addition, mysql and rabbitmq process should be exist (or in the same server or in another one, according to configuration).
+
+
+    UID        PID  PPID  C STIME TTY          TIME CMD
+    mysql        1     0  0 10:47 ?        00:00:00 mysqld
+
```
Comment (flopezag) [inline doc/admin.md]: blank line?

File: doc/admin.md [hunk 11]
```diff
@@ -49,4 +49,280 @@ Finally, we execute the commando package-import for the murano client library (c
...
+### Network interfaces Up & Open
+
+Taking into account the results of the ps commands in the previous
+section, we take the PID in order to know the information about the
+network interfaces up & open. To check the ports in use and listening,
+execute the command:
+
+    lsof -i | grep "$PID1\|$PID2"
+
+Where $PID1 and $PID2 are the PIDs of the tox processes obtained
```
Comment (flopezag) [inline doc/admin.md]: You write 2 PID, $PID1 and $PID2, but after you write 3 PIDs 762 and 821
+(redis-server) and 5604.

Apart from it, I think that it is a copy/paste from Policy Manager, rewrite it.

File: doc/admin.md [hunk 12]
```diff
@@ -49,4 +49,280 @@ Finally, we execute the commando package-import for the murano client library (c
...
+    murano-en 821 root    4u  IPv4 792083      0t0  TCP ac7d13082086:48475->rabbit:amqp (ESTABLISHED)
+
+
+### Databases
+
+
+The last step in the sanity check, once that we have identified the
+processes and ports is to check the database that have to be up and
+accept queries. For the first one, if we execute the following commands
+inside the code of Murano:
```
Comment (flopezag) [inline doc/admin.md]: inside the code of Murano?

File: doc/admin.md [hunk 13]
```diff
@@ -49,4 +49,280 @@ Finally, we execute the commando package-import for the murano client library (c
...
+the error is identified with these tests, the system admin will very
+often have to resort to more concrete and specific testing to pinpoint
+the exact point of error and a possible solution. Such specific testing
+is out of the scope of this section.
+
+### Resource availability
+
+The resource availability in the node should be at least 2Gb of RAM and
+8GB of Hard disk in order to prevent enabler’s bad performance in both
+nodes. This means that bellow these thresholds the enabler is likely to
```
Comment (flopezag) [inline doc/admin.md]: in both nodes? What does it mean?

File: doc/admin.md [hunk 14]
```diff
@@ -49,4 +49,280 @@ Finally, we execute the commando package-import for the murano client library (c
...
+experience problems or bad performance.
+
+###  Remote Service Access
+
```
Comment (flopezag) [inline doc/admin.md:240]: Copy/paste from Policy Manager...

File: doc/admin.md [hunk 15]
```diff
@@ -49,4 +49,280 @@ Finally, we execute the commando package-import for the murano client library (c
...
+organization with the following curl commands:
+
+    root@fiware:~# curl
+    -d '{"auth": {"tenantName": "<MY_ORG_NAME>",
+    "passwordCredentials":{"username": "<MY_USERNAME>", "password": "<MY_PASS>"}}}'
+    -H "Content-type: application/json" -H "Accept: application/xml"
+    http://<KEYSTONE_HOST>:<KEYSTONE_PORT>/v2.0/tokens
+
+The will be the name of my Organization/Tenant/Project predefined in the
+IdM GE (aka Keystone). The and variables will be the user name and
```
Comment (flopezag) [inline doc/admin.md]: "The and variables" something missing...

File: doc/admin.md [hunk 16]
```diff
@@ -49,4 +49,280 @@ Finally, we execute the commando package-import for the murano client library (c
...
+password predefined in the IdM GE and finally the and variables will be
```
Comment (flopezag) [inline doc/admin.md]: idem

File: doc/admin.md [hunk 17]
```diff
@@ -49,4 +49,280 @@ Finally, we execute the commando package-import for the murano client library (c
...
+the IP direction and port in which we can find the IdM GE (aka
+Keystone). This request should return one valid token for the user
+credentials together with more information in a xml format:
```
Comment (flopezag) [inline doc/admin.md]: Please use application/json, xml is not supported in FIWARE anymore I think and give very bad impression to put xml here.

File: doc/admin.md [hunk 18]
```diff
@@ -49,4 +49,280 @@ Finally, we execute the commando package-import for the murano client library (c
...
+
+    {
+        "packages":
+        [
+            ...
+        ]
+    }
+
+### Resource consumption
+No issues related to resources consumption have been detected.
```
Comment (flopezag) [inline doc/admin.md]: Really?

You need to write here which resources are you usually using in Murano, if I could remember just a top. And make differentiation between normal and high demand of Murano (the last one we usually gave an estimation of it).

Comment (hmunfru) [reply doc/admin.md]: mm I had a look at orion and monitoring and they don't specified that. But ok I can try that.

File: doc/admin.md [hunk 19]
```diff
@@ -49,4 +49,299 @@ Finally, we execute the commando package-import for the murano client library (c
...
+
+Characteristic   |   Low Usage        | High usage
+---------------  | -----------------  | -------------
+RAM              |  0.995 GB ~ 49%    |  1,4GB ~ 83,5%
+CPU              |  3.1% of a 2400MHz | 80% of a 2400MHz
+I/O HDD          |     6GB            |   6GB
+
+### I/O flows
+
+The Murano api is hearing from port 8082. Please refer to
```
Comment (jesuspg) [inline doc/admin.md]: hearing or listen?

File: doc/admin.md [hunk 20]
```diff
@@ -49,4 +49,299 @@ Finally, we execute the commando package-import for the murano client library (c
...
+    -d '{"auth": {"tenantName": "<MY_ORG_NAME>",
+    "passwordCredentials":{"username": "<MY_USERNAME>", "password": "<MY_PASS>"}}}'
+    -H "Content-type: application/json" -H "Accept: application/json"
+    http://<KEYSTONE_HOST>:<KEYSTONE_PORT>/v2.0/tokens
+
+The configuration variables will be the name of my Organization/Tenant/Project predefined in the
+Keystone, as well as the IP direction and port of the Keystone. This request should return one valid token for the user
+credentials together with more information in a json format:
+
+   {
```
Comment (jesuspg) [inline doc/admin.md]: bad formatting

**Comment by jesuspg** [comment]
LGTM

**Comment by flopezag** [comment]
LGTM

**Comment by flopezag** [comment]
LGTM

**Pull request merged**